packages/hash/api/src/auth/index.ts

```diff
@@ -99,7 +105,7 @@ const setupAuth = (params: {
   // eslint-disable-next-line @typescript-eslint/no-misused-promises -- https://github.com/DefinitelyTyped/DefinitelyTyped/issues/50871
   app.use(async (req, _res, next) => {
     const kratosSession = await publicKratosSdk
-      .toSession(undefined, req.header("cookie"))
+      .toSession({ cookie: req.header("cookie") })
```

Comment:
... continuation of the previous suggestion

```suggestion
      .toSession({
          cookie: req.cookies.join("; "),
          xSessionToken: sessionToken,
      })
```

Reply:
Applied with the above change :)
I used the `request` function instead for case insensitivity